src/PluginBase.php

```diff
@@ -24,6 +24,18 @@ abstract class PluginBase implements PluginInterface, EventSubscriberInterface
 
 	protected $io;
 
+	protected $configPath;
```

Comment:
Is there a reason why this property is `protected` and not `private`?

Reply:
Was just a hacky way to setup the path :/